_episodes/03-sql-joins.md

```diff
@@ -21,6 +21,17 @@ keypoints:
 To combine data from two tables we use the SQL `JOIN` command, which comes after
 the `FROM` command.
 
+Earlier, we talked about relational databases.
+
+Database tables are used to organize and group data by common characteristics or principles.  
+Often, we need to combine elements from separate tables into a single tables or queries for analysis and visualization.
+A JOIN is a means for combining columns from one (self-join) or more tables by using values common to each.
```

Comment:
I would leave out self-joins. They are not that much of a special case (you can use a single table multiple times even in the multiple-table case).